### PR TITLE
docs: rework vectorizer instructions

### PR DESCRIPTION
**Pull request by JamesGuthrie**
Makes the use-case of the different installation options more clear.

### CODE REVIEW

File: docs/vectorizer-worker.md [hunk 1]
```diff
@@ -31,49 +31,49 @@ To run vectorizer workers, you need to:
...
-             - ./data:/var/lib/postgresql/data
-         vectorizer-worker:
-           image: timescale/pgai-vectorizer-worker:v0.2.1
-           environment:
-             PGAI_VECTORIZER_WORKER_DB_URL: postgres://postgres:postgres@db:5432/postgres
-             OPENAI_API_KEY: <your-api-key>
-       ```
-
-   1. Replace the instances of `OPENAI_API_KEY` with a key from your AI provider.
+1. **Copy the following configuration into a file named `docker-compose.yml`**
```
Comment (adolsalamanca) [inline docs/vectorizer-worker.md:53]: Super nit and opinionated, but could skip it 😄 
```suggestion
1. **Copy the following configuration into a file named `docker-compose.yaml`**
```

**Review by adolsalamanca — APPROVED** [review]
Looks good, just left a nit

File: docs/vectorizer-worker.md (hunk 1 above)
Comment (JamesGuthrie) [reply docs/vectorizer-worker.md]: Done. Also TIL that they changed the name to `compose.yaml` in 2021 (https://stackoverflow.com/a/76752628). Old habits die hard.

File: docs/openai.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ This page shows you how to:
 
 ## Configure pgai for OpenAI
 
-Most pgai functions require an [OpenAI API key](https://platform.openai.com/docs/quickstart/step-2-set-up-your-api-key).
+To use the OpenAI functions, you will need an [OpenAI API key](https://platform.openai.com/docs/quickstart/step-2-set-up-your-api-key).
```
Comment (billy-the-fish) [inline docs/openai.md:11]:
```suggestion
To use the OpenAI functions, you need an [OpenAI API key](https://platform.openai.com/docs/quickstart/step-2-set-up-your-api-key).
```

File: docs/vectorizer-quick-start.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ If you prefer working with the OpenAI API instead of self-hosting models, you ca
 
 ## Setup a local development environment
 
-We use a docker-compose file to set up a development environment, it includes a:
+We use a docker compose file to set up a development environment, it includes a:
```
Comment (billy-the-fish) [inline docs/vectorizer-quick-start.md:8]:
```suggestion
To set up a development environment for OpenAI, use a docker compose file that includes a:
```

File: docs/vectorizer-worker.md [hunk 4]
```diff
@@ -31,49 +31,49 @@ To run vectorizer workers, you need to:
...
 
-The local developer environment is a docker configuration you use to develop and test pgai, vectorizers and vectorizer 
-worker locally. It includes a: 
-- Postgres deployment image with the TimescaleDB and pgai extensions installed
-- pgai vectorizer worker image  
+The end-to-end vectorizer worker is a batteries-included Docker Compose
+configuration which you use to test pgai, vectorizers and vectorizer worker
+locally. It includes a:
+- local Postgres instance with pgai installed,
+- embedding API service (Ollama)
```
Comment (billy-the-fish) [inline docs/vectorizer-worker.md:48]:
```suggestion
- Ollama API service embedding
```

File: docs/vectorizer-worker.md [hunk 5]
```diff
@@ -82,48 +82,25 @@ On your local machine:
...
-
 1. **Run the vectorizer worker**
 
    For self-hosted, you run a pgai vectorizer worker to automatically create embedding from the data in your
    database using [vectorizers you defined previously](/docs/vectorizer.md#define-a-vectorizer).
 
-   In a new terminal, start the vectorizer worker:
-   ```shell
-   docker compose up -d
+   Start the vectorizer worker:
```
Comment (billy-the-fish) [inline docs/vectorizer-worker.md:98]: Double-checking here. I seem to remember that the new terminal was necessary.

File: docs/voyageai.md [hunk 6]
```diff
@@ -8,7 +8,7 @@ This page shows you how to:
 
 ## Configure pgai for Voyage AI
 
-Most pgai functions require a [Voyage AI API key](https://docs.voyageai.com/docs/api-key-and-installation#authentication-with-api-keys).
+To use the Voyage AI functions, you will need a [Voyage AI API key](https://docs.voyageai.com/docs/api-key-and-installation#authentication-with-api-keys).
```
Comment (billy-the-fish) [inline docs/voyageai.md:11]:
```suggestion
To use the Voyage AI functions, you need a [Voyage AI API key](https://docs.voyageai.com/docs/api-key-and-installation#authentication-with-api-keys).
```

**Review by billy-the-fish — APPROVED** [review]
Couple of small TW nits (Always use the present tense :-) ). Good doc.

File: docs/vectorizer-worker.md (hunk 5 above)
Comment (JamesGuthrie) [reply docs/vectorizer-worker.md:98]: No longer

**Pull request merged**